### ISSUE TITLE
Free mobile API (France)

### ISSUE THREAD

**Issue by Neustradamus** [artprojectgroup/woocommerce-apg-sms-notifications#12]
Dear @artprojectgroup team,

It is possible to add the Free mobile SMS API support?

It is a free service for French customers which have a French free mobile number.

No e-mail address is needed.

Google search:
- https://www.google.com/search?q=freemobile+sms+api
- https://www.google.com/search?q=free+mobile+sms+api

GitHub:
- https://github.com/search?q=freemobile&type=repositories
- https://github.com/search?q="free+mobile"&type=repositories
- https://github.com/topics/freemobile
- https://github.com/topics/freemobilesms
- https://github.com/topics/free-mobile
- https://github.com/topics/free-mobile-operator
- https://github.com/topics/smsapi-free-mobile

Announcements (not a new service):
- https://www.freenews.fr/freenews-edition-nationale-299/free-mobile-170/nouvelle-option-notifications-par-sms-chez-free-mobile-14817
- https://www.universfreebox.com/article/26337/Nouveau-Free-Mobile-lance-un-systeme-de-notification-SMS-pour-vos-appareils-connectes

Example:
- "[https://smsapi.free-mobile.fr/sendmsg?user=XXXXXXXX&amp;pass=XXXXXXXXXXXXXX&amp;msg=Hello World ](https://smsapi.free-mobile.fr/sendmsg?user=XXXXXXXX&pass=XXXXXXXXXXXXXX&msg=Hello%20World%20)!"

Can you improve the SMS support?

Thanks in advance.

**Comment by artprojectgroup** [comment]
Please read the requirements at https://artprojectgroup.es/plugins-para-woocommerce/wc-apg-sms-notifications

**Comment by Neustradamus** [comment]
@artprojectgroup: Thanks for your answer, have you look my links to add the support?

**Comment by artprojectgroup** [comment]
Yes, but adding a platform is not free, so we have referred you to the requirements posted at https://artprojectgroup.es/plugins-para-woocommerce/wc-apg-sms-notifications.

Best regards.

**Comment by artprojectgroup** [comment]
Is it free only for France or internationally?

Create us a version with it integrated and we will see to integrate it for free in the next update.

Best regards,

**Comment by Neustradamus** [comment]
Free mobile customers can send on international numbers.

User only needs user and password and the message like it:
"[https://smsapi.free-mobile.fr/sendmsg?user=XXXXXXXX&pass=XXXXXXXXXXXXXX&msg=Hello World ](https://smsapi.free-mobile.fr/sendmsg?user=XXXXXXXX&pass=XXXXXXXXXXXXXX&msg=Hello%20World%20)!"

Can you add the support?
